explainer.md

```diff
@@ -1,751 +1,423 @@
-elements, which allows for transitions involving elements that are only in
-Page-A or only in Page-B.
+For the outgoing state, the following is captured for each part:
 
-Note that the order in which the transition elements are painted can be
-configured by UA and/or developer stylesheets using z-index.
+- An image of the element, including ink overflow if any. The browser may crop this to make efficient use of memory, and that cropping should be biased towards the viewport.
+- The layout dimensions.
+- A CSS transform that would place element from the layout viewport origin to its current quad.
+- An [`object-view-box`](https://drafts.csswg.org/css-images-4/#propdef-object-view-box) that coincides with the element's border box.
 
-- [Open question](https://github.com/WICG/shared-element-transitions/issues/23):
-  How should the default UA animation order these elements? And also handle a
-  change in associated elements between the 2 pages. Currently they are ordered
-  by paint order. For instance, parents are ordered before children, and
-  negative z-index elements are ordered before positive z-index elements.
+The above is also captured for the incoming state, although the image of the element is 'live', as in it's connected to the representation currently in the DOM.
 
-### How are transition elements painted?
+It's valid for some transition elements to only exist on one side of the DOM change.
 
-During the transition a new stacking context (called uber-root) is created with
-the following hierarchy :
+## Building the pseudo-element tree
+
+For each transitioning element (including the root), the following pseudo-element tree is created:
 
 ```
-uber-root stacking context
-├─ root stacking context
-└─ transition stacking context
+::page-transition-container(name)
+└─ ::page-transition-image-wrapper(name)
+   ├─ ::page-transition-outgoing-image(name)
+   └─ ::page-transition-incoming-image(name)
 ```
 
-This allows using the output of the root stacking context to provide a live root
-image for Page-B. The transition stacking context maps to the transition-root
-element. An alternate approach to this was to paint the transition-root in the
-[top layer](https://fullscreen.spec.whatwg.org/#top-layer). But that made it
-difficult to support transitions when there is other content in the top layer
-(fullscreen elements, dialog) and to ensure effects on the root element which
-are applied to the root stacking context (background-color, filter) are captured
-in the root image. See
-[issue](https://github.com/WICG/shared-element-transitions/issues/74) for
-detailed discussion.
+Where `name` is the `page-transition-tag` value, which is `root` for the root. If the transitioning element only existed on one side of the DOM change, then either the outgoing or incoming image will be missing.
+
+These trees are inserted into a `::page-transition` pseudo element, according to the render-order of their transition elements.
```

Comment:
```suggestion
These trees are inserted into a `::page-transition` pseudo element, according to the paint order of their transition elements.
```

Its a nit since we've used the word paint order in all other documentation.